mapper-processor/src/main/java/com/datastax/oss/driver/internal/mapper/processor/entity/EntityHelperGetMethodGenerator.java

```diff
@@ -62,24 +62,34 @@ public EntityHelperGetMethodGenerator(
     String returnName = "returnValue";
     getBuilder.addStatement("$1T $2L = new $1T()", returnType, returnName);
 
-    for (PropertyDefinition property : entityDefinition.getAllColumns()) {
+    for (PropertyDefinition property : entityDefinition.getAllValues()) {
       PropertyType type = property.getType();
       CodeBlock cqlName = property.getCqlName();
+      CodeBlock cqlResultName = property.getCqlResultName();
       String setterName = property.getSetterName();
       getBuilder.addCode("\n");
+      // if the result name is not the same as the name, it's likely this is a computed
```

Comment:
Not really happy with this, namely because throwing exceptions can be costly, but with `GettableByName` there is no way to determine if a definition is present without throwing an exception (`firstIndexOf` will throw an `IllegalArgumentException`).   If this was a `Row`, i could use `getDefinitions`.

Alternatively if we based alias names on property names (as described [here](https://github.com/tolbertam)) we could just remove this logic and assume the value is present, and if not fail like we would do for any other column.

Reply:
this is no longer an issue as this special logic is not needed.